tests/integration/external_deployment/test_external_deployment.py

```diff
@@ -339,5 +381,46 @@ def foo(self, docs, **kwargs):
         with f:
             response = f.post(on='/', inputs=d, target_executor='external_executor')
 
-
     assert response[0].text == 'external'
+
+
+@pytest.mark.parametrize('hosts', ['localhost,localhost', 'localhost'])
```

Comment:
External deployment and external replicas should be considered 2 different features and in my opinion tested differently in different test folders

Reply:
ok will refactor